Volume/Views/MainView and Tabs/BookmarksView.swift

```diff
@@ -106,50 +105,45 @@ struct BookmarksView: View {
 
     private var flyerContent: some View {
         VStack {
-            flyerSection(.upcoming)
-            flyerSection(.past)
+            if let upcomingFlyers = viewModel.upcomingFlyers,
+               let pastFlyers = viewModel.pastFlyers {
+
+                if pastFlyers.isEmpty && upcomingFlyers.isEmpty {
+                    noSavedContentView
+                } else if pastFlyers.isEmpty {
+                    flyerSection(.upcoming)
+                } else if upcomingFlyers.isEmpty {
+                    flyerSection(.past)
+                } else {
+                    flyerSection(.upcoming)
+                    flyerSection(.past)
+                }
+            }
         }
     }
 
     private func flyerSection(_ flyerSection: FlyerSection) -> some View {
         Section {
-            let selectedFlyers = flyerSection == .upcoming ? viewModel.upcomingFlyers : viewModel.pastFlyers
+            let selectedFlyers = (flyerSection == .upcoming) ? viewModel.upcomingFlyers : viewModel.pastFlyers
```

Comment:
Why add ")"? It's not swifty

Reply:
Tiffany said it was difficult to read at first glance, but I'll add it back.